### ISSUE TITLE
Feature Request - Echo the files without changing

### ISSUE THREAD

**Issue by vikas027** [piranha/goreplace#40]
As of now, `go-replace` replaces the strings in the files in the current directory, it would be nice to also print out the files without changing them.

For example, this is the current behaviour
`go-replace search -r replace`

A `sed` alternative of what I am looking for
`cat * | sed 's/search/replace/'`

**Comment by piranha** [comment]
Hah, it's funny, but I switched to ripgrep for searching, and for replacing of that kind, and use goreplace only for changing files...

**Comment by vikas027** [comment]
yeah, I know :) It will be just good to have the feature, I'll leave it up to you if you wanna build it. Thanks!
